### DISCUSSION TITLE
Can we compile JSX to Dioxus?

### DISCUSSION THREAD

**Discussion by Euraxluo** [DioxusLabs/dioxus#1499] [Ideas]
Can we compile JSX to Dioxus?

**Comment by Euraxluo** [comment]
or design a DSL to similar write dioxu

**Comment by ealmloff** [comment]
You can convert HTML to rsx with [rsx-rosetta](https://github.com/DioxusLabs/dioxus/tree/master/packages/rsx-rosetta) or if you want a more jsx-like macro you can use [dioxus-html-macro](https://github.com/DioxusLabs/dioxus-html-macro)
